doc/neps/nep-0044-restructuring-numpy-docs.rst

```diff
@@ -0,0 +1,234 @@
+===================================================
+NEP 44 — Restructuring the NumPy Documentation
+===================================================
+
+:Author: Ralf Gommers
+:Author: Melissa Mendonça
+:Author: Mars Lee
+:Status: Draft
+:Type: Process
+:Created: 2020-02-11
+
+Abstract
+========
+
+This document proposes a restructuring of the NumPy Documentation, both in form
+and content, with the goal of making it more organized and discoverable for
+beginners and experienced users.
+
+Motivation and Scope
+====================
+
+See `here <https://numpy.org/devdocs/>`_ for the front page of the latest docs.
+The organization is quite confusing and illogical (e.g. user and developer docs
+are mixed). We propose the following:
+
+- Reorganizing the docs into the four categories mentioned in [1]_;
+- Creating dedicated sections for Tutorials and How-Tos, including orientation
+  on how to create new content;
+- Adding an Explanations section for key concepts and techniques that require
+  deeper descriptions, some of which will be rearranged from the Reference Guide.
+
+Usage and Impact
+================
+
+The documentation is a fundamental part of any software project, especially
+open source projects. In the case of NumPy, many beginners might feel demotivated
+by the current structure of the documentation, since it is difficult to discover
+what to learn (unless the user has a clear view of what to look for in the
+Reference docs, which is not always the case).
+
+Looking at the results of a "NumPy Tutorial" search on any search engine also
+gives an idea of the demand for this kind of content. Having official high-level
+documentation written using up-to-date content and techniques will certainly
+mean more users (and developers/contributors) are involved in the NumPy
+community.
+
+Backward compatibility
+======================
+
+The restructuring will effectively demand a complete rewrite of links and some
+of the current content. Input from the community will be useful for identifying
+key links and pages that should not be broken.
+
+Detailed description
+====================
+
+As discussed in the article [1]_, there are four categories of doc content:
+
+- Tutorials
+- How-to guides
+- Explanations
+- Reference guide
+
+We propose to use those categories as the ones we use (for writing and
+reviewing) whenever we add a new documentation section.
+
+The reasoning for this is that it is clearer both for
+developers/documentation writers and to users where each information should go,
+and the scope and tone of each document. For example, if explanations are mixed
```

Comment:
I think this paragraph belabors a point that is evident.  Structuring the docs well is clearly a good idea.